lang/en/docs/tutorials/materials/specific/passivation-surface-silicon-surface.md

```diff
@@ -0,0 +1,184 @@
+adjusted_coordinate = [coord + delta for coord, delta in zip(second_si_atom_coordinate, [0.025, 0, 0.025])]
+print(f"Adjusted coordinate: {adjusted_coordinate}")
+
+new_coordinates = si_atoms_coordinates.copy()
+index_to_adjust = material.basis.coordinates.get_element_id_by_value(second_si_atom_coordinate)
+new_coordinates[index_to_adjust] = adjusted_coordinate
+material.set_coordinates(new_coordinates)
+```
+
+![Supercell Parameters](/images/tutorials/materials/passivation/passivation_surface_silicon/2-jl-setup-nb-adjust.webp "Supercell Parameters Visualization")
+
+### 1.4. Run Structure Adjustment
+
+Run the notebook using "Run > Run All Cells". This will:
+
+1. Load the Si(100) structure
+2. Adjust the position of the specified Si atom
+3. Create a supercell if specified in the parameters
+4. Visualize the adjusted structure
+
+![Adjusted Structure](/images/tutorials/materials/passivation/passivation_surface_silicon/3-wave-adjusted-material.webp "Adjusted Si(100) Structure")
+
+## 2. Passivate the Surface
+
+### 2.1. Open `passivate_slab.ipynb` Notebook
+
+Find and open the `passivate_slab.ipynb` notebook to add hydrogen atoms to the surface.
+
+### 2.2. Set Passivation Parameters
+
+Configure the following parameters for hydrogen passivation:
+
+```python
+# Passivation parameters
+PASSIVANT = "H"  # Chemical symbol for hydrogen
+BOND_LENGTH = 1.46  # Si-H bond length in Angstroms
+SURFACE = "top"  # Passivate only the top surface
+
+# Surface detection parameters
+SHADOWING_RADIUS = 1.8  # In Angstroms
+DEPTH = 0.5  # In Angstroms
+
+# Visualization parameters
+CELL_REPETITIONS_FOR_VISUALIZATION = [1, 1, 1]
+```
+
+Key parameters explained:
+
+- `BOND_LENGTH`: Si-H bond length from literature.
+- `SHADOWING_RADIUS`: Controls which atoms are considered surface atoms, set to be below the distance between top Si atoms pair.
+- `SURFACE`: Passivate only the top surface.
+- `DEPTH`: How deep to look for surface atoms, set to include only top Si atoms.
+
+![Passivation Parameters](/images/tutorials/materials/passivation/passivation_surface_silicon/4-jl-setup-nb-passivate.webp "Passivation Parameters Visualization")
+
+### 2.3. Run Passivation
+
+Run all cells in the notebook. The passivation process will:
+
+1. Detect surface Si atoms
+2. Add H atoms at the specified bond length
+3. Generate the passivated structure
+
+![Passivated Structure](/images/tutorials/materials/passivation/passivation_surface_silicon/5-jl-result-preview.webp "H-Passivated Si(100) Structure")
+
+## 3. Analyze Results
+
+After running both notebooks, examine the final structure:
+
+Check that:
+
+- The adjusted Si atom position is correct
+- Surface reconstruction is maintained
+- H atoms are properly placed above surface Si atoms
+
+![Final Structure](/images/tutorials/materials/passivation/passivation_surface_silicon/6-wave-result.webp "Final H-Passivated Si(100)")
+
+## 4. Save Passivated Structure
```

Comment:
This should be a generic item - similar to other notebooks - "Save the Results" or something like that, without mentioning the passivated structure